packages/@lwc/template-compiler/src/parser/index.ts

```diff
@@ -787,13 +833,10 @@ function applyLwcDirectives(
         ]);
     }
 
-    applyLwcExternalDirective(ctx, parsedAttr, element);
-    applyLwcDynamicDirective(ctx, parsedAttr, element);
-    applyLwcDomDirective(ctx, parsedAttr, element);
-    applyLwcInnerHtmlDirective(ctx, parsedAttr, element);
-    applyRefDirective(ctx, parsedAttr, element);
-    applyLwcSpreadDirective(ctx, parsedAttr, element);
-    applyLwcSlotBindDirective(ctx, parsedAttr, element);
+    // Bind LWC directives to element
+    for (const binding of LWC_DIRECTIVE_BINDINGS) {
+        binding(ctx, parsedAttr, element);
```

Comment:
bikeshedding but open to naming this something other than `binding`, I couldn't think of anything.